docs/preview/duckdb/backups_and_recovery.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: docu
+title: Backup and Recovery
+---
+DuckLake has two components: catalog and storage. The catalog contains all of DuckLake's metadata, while the storage contains all of the data files in Parquet format. The catalog is a [database]({% link docs/preview/duckdb/usage/choosing_a_catalog_database.md %}), while the storage layer can be any [filesystem backend supported by DuckDB]({% link docs/preview/duckdb/usage/choosing_storage.md %}). These two components have different backup strategies, so this document will address them separately.
+
+> In this document, we will focus on disasters caused by human errors or application failures/malfunctions that result in data corruption or loss.
+
+## Catalog Backup and Recovery
+
+Backup and recovery strategies depend on the SQL database you are using as a DuckLake catalog.
+
+### DuckDB Catalog
+
+For DuckDB, the best approach is to perform regular backups of the metadata database. If the original database is corrupted, tampered with, or even deleted, you can recover from this backup.
+
+```sql
+-- Backup
+ATTACH 'main.db' AS main (READ_ONLY);
+ATTACH 'backup.db' AS backup;
+COPY DATABASE FROM main TO backup;
```

Comment:
```suggestion
BEGIN TRANSACTION;
ATTACH 'main.db' AS main (READ_ONLY);
ATTACH 'backup.db' AS backup;
COPY FROM DATABASE main TO backup;
DETACH main;
DETACH backup;
COMMIT;
```

Reply:
(this needs to be tested, nut currently I think you COPY FROM DATABASE (invert) is not atomic, so wrapping all in transaction should all work just in case.